applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -14,20 +14,189 @@
 #include "cJSON_os.h"
 #include "cloud_codec.h"
 
+#include <logging/log.h>
 
-static const char * const sensor_type_str[] = {
-	[CLOUD_SENSOR_GPS] = "GPS",
-	[CLOUD_SENSOR_FLIP] = "FLIP",
-	[CLOUD_SENSOR_BUTTON] = "BUTTON",
-	[CLOUD_SENSOR_TEMP] = "TEMP",
-	[CLOUD_SENSOR_HUMID] = "HUMID",
-	[CLOUD_SENSOR_AIR_PRESS] = "AIR_PRESS",
-	[CLOUD_SENSOR_AIR_QUAL] = "AIR_QUAL",
-	[CLOUD_LTE_LINK_RSRP] = "RSRP",
-	[CLOUD_DEVICE_INFO] = "DEVICE",
+LOG_MODULE_REGISTER(cloud_codec);
```

Comment:
I don't see any logger calls added, could this be removed? If you want to keep it though, you should provide it with a specific log level.

Reply:
I'll remove it.